Sources/CartonCLI/Commands/Bundle.swift

```diff
@@ -42,6 +46,12 @@ struct Bundle: AsyncParsableCommand {
   @Flag(help: "Emit names and DWARF sections in the .wasm file.")
   var debugInfo: Bool = false
 
+  @Option(
+    name: .long,
+    help: "Which optimizations to apply to the .wasm binary output.\n Available values: \(WasmOptimizations.allCases.map { $0.rawValue }.joined(separator: ", "))"
```

Comment:
Still wondering if multiline string would allow us to use multiline interpolation here?

```suggestion
    help: """
      Which optimizations to apply to the .wasm binary output.\n Available values: \(
        WasmOptimizations.allCases.map { $0.rawValue }.joined(separator: ", ")
      )
    """
```

Reply:
The latest commit builds, is that what you had in mind?